fiasko_bro/code_validator.py

```diff
@@ -146,6 +146,15 @@ def run_validator_group(self, group, add_warnings=False, *args, **kwargs):
             return errors
         return errors
 
+    def get_syntax_errors_in_repos(self, solution_repo, original_repo=None):
+        syntax_errors = validators.has_no_syntax_errors(solution_repo)
+        if syntax_errors:
+            return [syntax_errors]
+        if original_repo:
```

Comment:
Actually, we don't care if the `original_repo` contains any syntax errors, we never use its syntax tree (and we don't throw the exception while analyzing it: https://github.com/devmanorg/fiasko_bro/blob/master/fiasko_bro/repository_info.py#L53).

It's actually a mistake that we create the tree for the original repo at all. We are going to fix it in `0.0.3`, but for now, let's just ignore it if `original_repo` contains any syntax errors.